.github/workflows/tox.yml

```diff
@@ -27,7 +28,19 @@ jobs:
       run: |
         pip install tox
 
-    - name: Run Test
+    - name: Run Windows Test
+      if: runner.os == 'Windows'
+      run: |
+        python -m tox -c python/tox_windows.ini
+
+    - name: Run Linux Test
+      if: runner.os != 'Linux'
+      run: |
+        cd $GITHUB_WORKSPACE/python
+        tox -e py
+
+    - name: Run MacOS Test
+      if: runner.os != 'macOS'
```

Comment:
I don't think separate rules are necessary for each OS. The same test works on all of them.